webapp/ordering/models.py

```diff
@@ -79,6 +79,13 @@ class Meta:
         help_text="Whether we've sent order reminders to our members"
     )
 
+    distribution_coordinator = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        models.SET_NULL,
+        null=True,
+        related_name="coordinating_distribution"
```

Comment:
This will be the name of the reverse relation on `VokoUser` to fetch all order rounds where the user was distribution coordinator. I'm not sure the current value make sense. I propose `VokoUser.coordinating_orderrounds`.

Reply:
Good point, but it's a specific kind of coordination, so I'm proposing `coordinating_distribution_orderrounds`.